README.md

```diff
@@ -80,5 +80,17 @@ Once CMake has generated the build files, within `<WorkSpace Dir>/build` run...
 - [Build] `cmake --build .` or `ninja -v` (if the build generator was Ninja)
 - [Test] `ctest --parallel`
 
+Documentation

```

Comment:
Let's add a description of what this documentation is. Something like:

Documentation for each build is automatically generated from the code using Doxygen(link here). Documentation for releases and recent builds can be viewed via the links that follow.